src/utils.test.js

```diff
@@ -0,0 +1,62 @@
+import * as requests from "./utils/utils";
+
+describe("Utils", () => {
+	let url = `http://localhost:1234/`;
+
+	it("takes in a date and returns the time and date in an array", () => {
+		let date = "2021-11-16T00:37:01.581Z";
+
+		expect(requests.sortDateTime(date)[0]).toBe("2021-11-16");
+		expect(requests.sortDateTime(date)[1]).toBe("00:37:01");
+	});
+
+	it("takes in a date and returns the time and date in an array", () => {
```

Comment:
this description doesn't match the test.

Reply:
yes, thanks ! I spotted dit with Elle today and corrected it 😄